web/src/components/ui/button.tsx

```diff
@@ -6,7 +6,7 @@ import { cn } from "@/src/utils/tailwind";
 import { Loader2 } from "lucide-react";
 
 const buttonVariants = cva(
-  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:opacity-50",
+  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50",
```

Comment:
strong preference for cursor-none (this was the default of this shadcn/ui button) to indicate that it is disabled

Reply:
I believe in this case you wouldn't see the title tooltip?

Reply:
good catch, reenabled pointer-events when disabled for this button and added cursor-not-allowed as title takes some time to show up: https://github.com/langfuse/langfuse/commit/cc08408f0a9d67ff27276e73819125c1e6c53307